tests/influence/test_influences.py

```diff
@@ -413,3 +476,15 @@ def test_influences_arnoldi(
     )
 
     assert np.allclose(direct_influence, low_rank_influence, rtol=1e-1)
+
+
+@pytest.mark.torch
+@pytest.mark.parametrize("run", [1, 2])
+def test_deterministic_seed(deterministic_seed, run):
```

Comment:
is this test really needed? It tests PyTorch's behaviour which is something that we do not control anyway.

Reply:
Yeah, agree, will remove it